.github/workflows/examples.yml

```diff
@@ -60,13 +64,8 @@ jobs:
     - name: Seismic elastic examples
       shell: bash -l {0}
       run: |
-        py.test examples/seismic/elastic/elastic_example.py
-        py.test examples/seismic/viscoelastic/viscoelastic_example.py
-
-    - name: Seismic viscoacoustic examples
-      shell: bash -l {0}
-      run: |
-        py.test examples/seismic/viscoacoustic/viscoacoustic_example.py
+        python examples/seismic/elastic/elastic_example.py
+        python examples/seismic/viscoelastic/viscoelastic_example.py
```

Comment:
This is a big expensive to execute compared to py.test.

Reply:
This CI runs in ~8min, still way faster than pytest so doesn't really matter

Reply:
ok, then. I am fine anyway.

Reply:
are errors caught as expected even w/o pytest ? 

pytest was gonna run `test_...` which has asserts inside

Reply:
All tests are run above......

Reply:
if nothing can fail, why are we running these? for segfaults?

Reply:
Yeah these are 3D test is for 2D so making sure 3D doesn't error